explorer/client/src/components/search/Search.tsx

```diff
@@ -12,10 +12,11 @@ type SearchCategory = 'all' | 'transactions' | 'addresses' | 'objects';
 function getPlaceholderText(category: SearchCategory) {
     switch (category) {
         case 'addresses':
-            return 'Search by address';
+            return 'Search by Address';
         case 'transactions':
-            return 'Search by tx ID';
+            return 'Search by TxId';
```

Comment:
i know this is how we format it in other parts of the code, but i feel like it's not very user friendly label & `tx ID` reads more naturally.